motionplan/motionPlanner.go

```diff
@@ -526,7 +526,7 @@ func CheckPlan(
 	if sfPlanner.planOpts, err = sfPlanner.plannerSetupFromMoveRequest(
 		currentPosition,                    // starting pose
 		planNodes[len(planNodes)-1].Pose(), // goalPose
-		plan[0],                            // starting configuration
+		sf.sliceToMap(currentInputs),       // starting configuration
```

Comment:
[note] This fix is based on a PR @nfranczak spun up a few weeks ago. Peter suggested a move-involved solution. I would like to push this fix for now along with the comment I added saying to replace it when bandwidth permits

Reply:
Original PR: https://github.com/viamrobotics/rdk/pull/3123